Parsers/ASimAuditEvent/Parsers/ASimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,183 @@
+            | parse description_s with "Parameters updated for " temp_config2: string " (ID: " temp_configid2: string ") for policy with ID " temp_policyid2: string
+            | extend
+                temp_operation = coalesce(temp_config1, temp_config2),
+                temp_configid = coalesce(temp_configid1, temp_configid2)
+            | extend
+                EventType = "Set", 
+                Operation = strcat("Parameters updated for ", temp_operation, " for policy ", temp_policyname1, tostring(split(temp_policyid2, "{")[0])),
+                Object = strcat("Policy ", coalesce(temp_policyname1, temp_policyid2)),
+                ObjectType = "Policy Rule",
+                AdditionalFields = bag_pack("config id", temp_configid);
+        let Reputation = allData
+            | where description_s has_cs "Reputation"
+            | parse description_s with "User " * " " temp_type1: string " Reputation" * " for Organization ID " temp_orgid1: string " of type " temp_reptype1: string " to " temp_list1: string " with content: " temp_content1: string " | " temp_restmessage1: string
+            | parse description_s with "User " * " " temp_type2: string " Reputation" * " for Organization ID " temp_orgid2: string ": " temp_content2: string " | " temp_restmessage2: string
+            | extend
+                temp_type = coalesce(temp_type1, temp_type2),
+                Object = iff(isnotempty(temp_reptype1), strcat("Reputation Override of type ", temp_reptype1), "Reputation Override"),
+                ObjectType = "Configuration Atom"
+            | lookup EventTypeLookup on temp_type
+            | extend Operation = strcat(temp_type, " ", Object)
+            | extend AdditionalFields = bag_pack("reputation value", coalesce(temp_content1, temp_content2));
+        union
+            Enabled,
+            Set,
+            AlertNotify,
+            CustomRole,
+            Policy,
+            Changed,
+            ParamsUpdated,
+            Reputation
+        | extend
+            EventStartTime = unixtime_milliseconds_todatetime(eventTime_d),
+            EventSeverity = coalesce(EventSeverity1, "Informational"),
+            AdditionalFields = bag_merge(AdditionalFields, bag_pack("flagged", flagged_b))
+        | extend
+            EventProduct = "Carbon Black Cloud",
+            EventSchema = "AuditEvent",
+            EventSchemaVersion = "0.1",
+            EventVendor = "VMware",
+            EventResult = "Success",
+            DvcAction = "Allow",
+            EventCount = int(1)
+        | project-rename
+            ActorUsername = loginName_s,
+            EventUid = _ItemId,
+            SrcIpAddr = clientIp_s,
+            EventMessage = description_s,
+            EventOriginalUid = eventId_g,
+            EventOwner = orgName_s
```

Comment:
can we map this with TargetDvcScope?
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-audit

Reply:
orgName_s represents the name of the organization in which a user exists who has performed this audit activity in VMware Carbon Black Cloud. And EventOwner field depicts "The owner of the event, which is usually the department or subsidiary in which it was generated". So we thought to map it with EventOwner but we can map it with ActorScope instead of TaretDvcScope.

Reply:
@niralishah-crest thanks